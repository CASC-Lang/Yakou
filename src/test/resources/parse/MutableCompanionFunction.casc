class MutableCompanionFunction

impl MutableCompanionFunction {
    mut fn a1() {}
}