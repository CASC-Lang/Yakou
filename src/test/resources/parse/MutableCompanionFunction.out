error: Companion function cannot be mutable
--> ./MutableCompanionFunction.casc:4:4
3 | impl MutableCompanionFunction {
4 |     mut fn a1() {}
        ^^^= hint: Remove this `mut` keyword
5 | }